### PR TITLE
8340437: Open source few more AWT Frame related tests 

### PR DESCRIPTION
**Pull request by kumarabhi006**
AWT Frame related tests are converted from applet to manual/automated and moved to open.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8340437](https://bugs.openjdk.org/browse/JDK-8340437): Open source few more AWT Frame related tests (**Bug** - P4)

### Reviewers
 * [Alexander Zuev](https://openjdk.org/census#kizune) (@azuev-java - **Reviewer**) Review applies to [837c6d1b](https://git.openjdk.org/jdk/pull/21199/files/837c6d1b5cf80b85c01737457e213dbfae733eaf)
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/21199/head:pull/21199` \
`$ git checkout pull/21199`

Update a local copy of the PR: \
`$ git checkout pull/21199` \
`$ git pull https://git.openjdk.org/jdk.git pull/21199/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 21199`

View PR using the GUI difftool: \
`$ git pr show -t 21199`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/21199.diff">https://git.openjdk.org/jdk/pull/21199.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/21199#issuecomment-2376241473)

### CODE REVIEW

**Review by azuev-java — APPROVED** [review]
Looks good.

File: test/jdk/java/awt/Frame/MultiScreenTest.java [hunk 1]
```diff
@@ -0,0 +1,484 @@
...
+    static GraphicsEnvironment ge;
+    static GraphicsDevice[] gs;
+
+    public static void main(String[] args) throws Exception {
+        ge = GraphicsEnvironment.getLocalGraphicsEnvironment();
+        gs = ge.getScreenDevices();
+        if (gs.length < 2) {
+            throw new SkippedException("You have only one monitor in your system - test passed");
+        }
+        MultiScreenTest obj = new MultiScreenTest();
```
Comment (prrace) [inline test/jdk/java/awt/Frame/MultiScreenTest.java:78]: Although this test will bail if there aren't two monitors, I'd like to see us remember to add @key multimon to such tests.
Then if someone wants to run multimon testsand ONLY those tests they can specify this key 
I'm sure we missed a couple of opportunities to do it already but let's not miss this one.

See 
https://github.com/openjdk/jdk/blob/master/test/jdk/TEST.ROOT

File: test/jdk/java/awt/Frame/IMStatusBar.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+ * @summary Checks that IMStatusBar does not affect Frame layout
+ * @run main/manual/othervm -Duser.language=ja -Duser.country=JP IMStatusBar
+ */
+
+public class IMStatusBar {
+
+    public static void main(String[] args) throws Exception {
+        String INSTRUCTIONS = """
+                If the window appears the right size, but then resizes so that the
+                status field overlaps the bottom label, press Fail; otherwise press Pass.
```
Comment (prrace) [inline test/jdk/java/awt/Frame/IMStatusBar.java:44]: does this ALWAYS bring up an IM window ? On all platforms.

File: test/jdk/java/awt/Frame/MultiScreenTest.java (hunk 1 above)
Comment (kumarabhi006) [reply test/jdk/java/awt/Frame/MultiScreenTest.java:79]: Ok.. added ` @key multimon `.

File: test/jdk/java/awt/Frame/IMStatusBar.java (hunk 2 above)
Comment (kumarabhi006) [reply test/jdk/java/awt/Frame/IMStatusBar.java:44]: I checked on all platforms (Win, Linux and Mac), IM window always shown.

**Review by prrace — APPROVED** [review]

**Pull request closed without merging**